app/routers/exercise.py

```diff
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, Depends, Request
+from app.database.models import User
+from app.database.database import get_db
+from app.dependencies import templates
+from app.routers.user_exercise import get_user_exercise
+from datetime import datetime
+from app import config
+
+router = APIRouter(
+    prefix="/exercise",
+    tags=["exercise"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("/")
+async def exercise(
+        request: Request,
+        session=Depends(get_db)
+        ):
+    """
+    If is active exercise = True
+    Show user exercise for specific day
+    """
+    user = session.query(User).filter_by(id=1).first()
+    if not user:
```

Comment:
I remember I saw creating a default user in other places in code.
Maybe consider extract that to a util or internal function, so all can get or create default user?